integration/benchnet2/Makefile

```diff
@@ -1,11 +1,33 @@
+# default values that can callers override when calling target
+ACCESS_NODES=2
+COLLECTION_NODES=6
+CONSENSUS_NODES=3
+EXECUTION_NODES=2
+VERIFICATION_NODES=1
+
 gen-bootstrap:
-	go run -tags relic ../../cmd/bootstrap genconfig --address-format "%s%d.${NAMESPACE}:3569" --access 2 --collection 6 --consensus 3 --execution 2 --verification 1 --weight 1000 -o ./ --config ./bootstrap/conf/node-config.json
+	go run -tags relic ../../cmd/bootstrap genconfig --address-format "%s%d.${NAMESPACE}:3569" --access $(ACCESS_NODES) --collection $(COLLECTION_NODES) --consensus $(CONSENSUS_NODES) --execution $(EXECUTION_NODES) --verification $(VERIFICATION_NODES) --weight 1000 -o ./ --config ./bootstrap/conf/node-config.json
```

Comment:
You would likely need to match `-nclusters` with the `CONSENSUS_NODES`

Reply:
As discussed, will leave this check out for now since bootstrap logic is always using 2 clusters which requires a min of 6 collection nodes and there's already a check for min number of collection nodes.